src/.clang-tidy

```diff
@@ -6,6 +6,8 @@ modernize-use-default-member-init,
 modernize-use-nullptr,
 readability-redundant-declaration,
 readability-redundant-string-init,
+performance-for-range-copy,
+performance-unnecessary-copy-initialization,
```

Comment:
nit: these were in alphabetical order before, it helps visual search

Reply:
fixed thanks